### DISCUSSION TITLE
RandSimulateLowResolutiond with different default parameters than in nnUNet

### DISCUSSION THREAD

**Discussion by jmlipman** [Project-MONAI/MONAI#7845] [Q&A]
This [transformation](https://docs.monai.io/en/stable/transforms.html#monai.transforms.RandSimulateLowResolutiond) is a "Random simulation of low resolution corresponding to nnU-Net’s SimulateLowResolutionTransform". However, in nnUNet (in [batchgenerators](https://github.com/MIC-DKFZ/batchgenerators/blob/master/batchgenerators/augmentations/resample_augmentations.py#L26) actually), downsampling is done linearly and upsampling via nearest neighbor, whereas in MONAI is the opposite. I was wondering if there is a reason behind this discrepancy.

Thanks!

**Comment by KumoLiu** [comment]
Hi @jmlipman, thanks for your interest here.
It was just a default value. https://github.com/Project-MONAI/MONAI/blob/96bfda00c6bd290297f5e3514ea227c6be4d08b4/monai/transforms/spatial/array.py#L3470
You can update it when you initialize the transform.

Thanks.
